README.md

```diff
@@ -117,13 +117,19 @@ If you're using more then one settings file change manage.py to point to local o
 $ python manage.py runserver
 ```
 
-This will run the server on http://127.0.0.1:8000. You can configure the host and port as needed.
+This will run the server on http://127.0.0.1:8000 (Don't open the link in your browser yet). You can configure the host and port as needed.
 
 ### Create an activity user
 
 Once you have created your user account, you need to create an `activity user` that is linked to this user account.
 
 Go to http://127.0.0.1:8000/admin and sign in using your superuser account. Under the `workflow` model, you'll find `activity user`. Create a new activity user making sure you associate your user under the `user` attribute.
+### Open the dashboard
```

Comment:
Do we need this instruction for a user who has already created Activity User? I doubt

Reply:
This only happens during the first time setup. It's weird issue that we can probably address later.